packages/peregrine/lib/List/__tests__/useListState.spec.js

```diff
@@ -1,197 +0,0 @@
-import React from 'react';
```

Comment:
I personally feel we need to get this back or a version of this if not exactly the same for separation of concerns reasons to put functionality into one file and presentation into other.

Reply:
I can be convinced 😄

Reply:
The reason being, if tomorrow we want to change the look of the `List` or `Item`, or if we want to add another type of selection to the existing `radio` and `checkbox`, it will be easy to perform.